### DISCUSSION TITLE
Custom checkout & shipping page

### DISCUSSION THREAD

**Discussion by MaximBudnik** [swellstores/community#182] [Help] [answered]
We currently have a shop that already uses Swell. But when the user wants to finish the order, he is redirected to the page that is managed by Swell `https://<shopDomain>.swell.store/checkout/<orderId>/shipping`. But we want to handle shipping & payment forms directly on our website. 
Is there a way to implement this case?

**Comment by mustafaswell** [comment, marked as answer]
Hi @MaximBudnik, 

You can enable Custom Checkout by going into _Settings > Storefront > Custom checkout_

Let me know if you have further issues!

**Reply by MaximBudnik** [reply]
Hi @mustafaswell,
Do you have code examples of custom checkout?

**Reply by sags95** [reply]
@MaximBudnik We currently have this guide here that covers mainly the logic around payment elements with Stripe https://developers.swell.is/guides/render-stripe-elements-with-react
